Makefile

```diff
@@ -252,16 +252,19 @@ else
 buildx:
 endif
 
+BUILDX_COMMIT ?= v0.3.0
+export BUILDX_COMMIT
+
 bundles/buildx: BUILDX_DOCKERFILE ?= Dockerfile.buildx
-bundles/buildx: BUILDX_COMMIT ?= v0.3.0
 bundles/buildx: bundles ## build buildx CLI tool
 	# This intetionally is not using the `--output` flag from the docker CLI which is a buildkit option
 	# The idea here being that if buildx is being used, it's because buildkit is not supported natively
-	docker build -f $(BUILDX_DOCKERFILE) -t "moby-buildx:$(BUILDX_COMMIT)" \
+	docker build -f $(BUILDX_DOCKERFILE) -t "moby-buildx:$${BUILDX_COMMIT:-latest}" \
 		--build-arg BUILDX_COMMIT \
```

Comment:
So, why not just use `--build-arg BUILDX_COMMIT=$(BUILDX_COMMIT)` instead?

Reply:
If possible, I like to have the build-args take their value from the environment. That way if no env-var is set, the value from the Dockerfile is leading. Putting the value _after_ means if it's not set, it will be `--build-arg=`, which is equivalent to `--build-arg=""`, which will override the value in the Dockerfile

Reply:
ok, makes sense